### PR TITLE
Add kGREYConfigKeySpeedUpAnimations

### LINKED ISSUE #102: Add helper method to speed up animations in tests.

### PR DESCRIPTION
**Pull request by bootstraponline**
Fix #102

### CODE REVIEW

File: EarlGrey/Additions/CALayer+GREYAdditions.m [hunk 1]
```diff
@@ -73,6 +73,11 @@ - (void)grey_adjustAnimationToAllowableRange:(CAAnimation *)animation {
     return;
   }
 
+  CFTimeInterval layerSpeed =
```
Comment (khandpur) [inline EarlGrey/Additions/CALayer+GREYAdditions.m]: Let's talk about this on Slack. I might have jumped the gun by suggesting this too early. Ping me whenever you're available.

Comment (bootstraponline) [inline EarlGrey/Additions/CALayer+GREYAdditions.m]: Sounds good.

**Comment by bootstraponline** [comment]
Discussed on slack, decided to move to a method instead of a config option. Closing for now until it's decided where to place the method.

**Comment by bootstraponline** [comment]
https://github.com/google/EarlGrey/pull/124

**Pull request closed without merging**